### PR TITLE
1.0.0 docs

### PR DESCRIPTION
**Pull request by pjfanning**
* including job to publish docs
* release notes

### CODE REVIEW

File: docs/src/main/paradox/release-notes.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Release Notes
+
+## 1.0.0
+Apache Pekko Persistence Cassandra 1.0.0 is based on Akka Persistence Cassandra 1.0.6. Pekko came about as a result of Lightbend's decision to make future
+Akka releases under a [Business Software License](https://www.lightbend.com/blog/why-we-are-changing-the-license-for-akka),
+a license that is not compatible with Open Source usage.
+
+Apache Pekko has changed the package names, among other changes. The new packages begin with `org.apache.pekko.persistence.cassandra` instead of `akka.persistence.cassandra`.
+
+Config names have changed to use `pekko` instead of `akka` in their names.
```
Comment (nvollmar) [inline docs/src/main/paradox/release-notes.md:10]: Maybe also mention here that the default keyspaces have changed to `pekko` and `pekko_snapshot`.

File: docs/src/main/paradox/release-notes.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+Users switching from Akka to Pekko should read our [Migration Guide](https://pekko.apache.org/docs/pekko/current/project/migration-guides.html).
```
Comment (nvollmar) [inline docs/src/main/paradox/release-notes.md:12]: Maybe also reference the projects migration guide under docs/src/main/paradox/migrations.md

**Review by nvollmar — APPROVED** [review]
lgtm

**Review by He-Pin — APPROVED** [review]
Lgtm

**Pull request merged**